.env.sample

```diff
@@ -5,15 +5,43 @@ OKTA_ISSUER="_GET_THIS_OKTA_"
 OKTA_ORG_URL="_GET_THIS_OKTA_"
 OKTA_OIDC_REDIRECT_URI="http://localhost:8666/authorization-code/callback"
 OKTA_API_TOKEN="_GET_THIS_OKTA_"
+OKTA_SIW_VERSION="Defaults to v5.16.1. See https://github.com/okta/okta-signin-widget/releases for versions"
+OKTA_SIW_HASH="sha384-a2yCLiNNwZ2BjqZF19qFKM2XWAuGKREPL8S3FckTr8+S4JDdpNqPehJUjVfNHf5a"
+OKTA_AUTH_JS_VERSION="Defaults to v5.11.0. See https://github.com/okta/okta-auth-js/releases for versions"
+OKTA_AUTH_JS_HASH="sha384-Wk5gwb+o8f4psT7pFZVAGgGprt1nIg88L4BOLYnaIe3fvxdS+5u7xP/n8xuqMWFr"
```

Comment:
@mdwallick Not sure I agree with making this Dynamic. I can't see a use-case where we want at run time to change versions in both SIW and AuthJS. Plus there is additional context Contributors need to understand and feel it a little over-engineered. My thoughts are that if you going to go up or down version there is still JS code that you have to modify that is associated with SIW/AuthJS version. ie. `feature: ...`

Let me know if you discuss this further.

Reply:
Perhaps this is a little overdone, but I was erring on the side of making SonarCloud happy. The idea is that for most end users (not contributors), they're going to use whatever the default version/hash of the SIW and AuthJS is used if no explicit environment variable is set. The goals for this change were to remove the use of the S3 CDN, give one the option of specifying a specific version (if they need to) and consolidating all of the various references to the SIW and AuthJS libraries. I'm happy to pull out the hash thing if you think it's not necessary. Like I said, it was about making the SonarCloud tests pass.

Reply:
I removed the hash thing. Please approve if you have no other issues. Thanks!